packages/wonder-blocks-search-field/src/components/search-field.tsx

```diff
@@ -72,6 +72,10 @@ type Props = AriaProps & {
      * Called when a key is pressed.
      */
     onKeyDown?: (event: React.KeyboardEvent<HTMLInputElement>) => unknown;
+    /**
+     * Called when a key is released.
+     */
+    onKeyUp?: (event: React.KeyboardEvent<HTMLInputElement>) => unknown;
```

Comment:
question: I noticed we often deconstruct the specific props before applying it to the underlying element. Do we do that so that the remaining `otherProps` will usually be the aria props? Are there any drawbacks to letting other props be part of the `otherProps`?

Here's an example where we do that with the other event handlers:
https://github.com/Khan/wonder-blocks/blob/dd2759e5f5c8bc5016cee0773770987a71b28cac/packages/wonder-blocks-search-field/src/components/search-field.tsx#L126-L130

Reply:
That's correct, we use that for aria props, and also for letting other valid attributes be injected in the inner node. I don't think there are drawbacks with that logic.... the only thing is that TS would complain, but this can be fixed by adding the missing props, like the one included here :).